lib/solaar/ui/window.py

```diff
@@ -115,17 +109,17 @@ def _status_line(label_text):
         b = Gtk.Box.new(Gtk.Orientation.HORIZONTAL, 8)
         b.set_size_request(10, 28)
 
-        b._label = Gtk.Label(label_text)
-        b._label.set_alignment(0, 0.5)
+        b._label = Gtk.Label(label=label_text)
+        # deprecated - not needed  b._label.set_alignment(0, 0.5)
```

Comment:
Not needed? Then remove it before it is merged.

Reply:
Agreed.   That's in the deprecation PR, which I will change.